activation/wire/wire_v1.go

```diff
@@ -0,0 +1,240 @@
+package wire
+
+import (
+	"fmt"
+
+	"github.com/spacemeshos/go-spacemesh/codec"
+	"github.com/spacemeshos/go-spacemesh/common/types"
+	"github.com/spacemeshos/go-spacemesh/hash"
+)
+
+//go:generate scalegen
+
+type ActivationTxV1 struct {
+	InnerActivationTxV1
+
+	SmesherID types.NodeID
+	Signature [64]byte
```

Comment:
I think this could be

```suggestion
	Signature types.EdSignature
```

`types.EdSignature` is a `[64]byte` but giving it a concrete name prevents other `[64]bytes` from being accidentally used as signatures